### PR TITLE
BigAnimal: CLI updates for 3.1

### PR DESCRIPTION
**Pull request by francoughlin**
## What Changed?

Updated BigAnimal PGD CLI commands for version 3.1

### CODE REVIEW

**Review by drothery-edb — APPROVED** [review]

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 1]
```diff
@@ -334,7 +334,11 @@ dataGroups:
       array_nulls: true                        
     backupRetentionPeriod: 30d
 witnessGroups:
-  - region: uksouth
+  - provider: azure
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:337]: both data groups must also have `customMaintenanceWindow` fields defined in the example

**Review by edbamolvyavahare — CHANGES_REQUESTED** [review]
- both data groups must also have customMaintenanceWindow fields defined in the example
- Rename High Availability -> Primary/Standby High Availability at all places

**Comment by nidhibhammar** [comment]
@edbamolvyavahare, please review the updates and let us know if any further changes are required or we are good to publish?

**Review by edbamolvyavahare — APPROVED** [review]
LGTM

**Pull request merged**